projects/dsp-ui/src/lib/action/directives/external-links/external-links.directive.ts

```diff
@@ -0,0 +1,39 @@
+import { isPlatformBrowser } from '@angular/common';
+import { Directive, HostBinding, Inject, Input, OnChanges, PLATFORM_ID } from '@angular/core';
+
+@Directive({
+  selector: 'a[href]'
+})
+export class ExternalLinksDirective implements OnChanges {
+
+    @HostBinding('attr.rel') relAttr = '';
+    @HostBinding('attr.target') targetAttr = '';
+    @HostBinding('attr.href') hrefAttr = '';
+    @HostBinding('style.color') fontColor = 'primary';
+    @Input() href: string;
```

Comment:
Variable declaration order should be: @Inputs, @Outputs, @ViewChild, @HostBinding

Reply:
you are right
done in 5a176c2